assets/common/sidecars/controller_driver_kube_rbac_proxy.yaml

```diff
@@ -6,7 +6,7 @@ spec:
   template:
     spec:
       containers:
-        - name: kube-rbac-proxy-${LOCAL_METRICS_PORT}
+        - name: driver-kube-rbac-proxy
```

Comment:
Have you checked this doesn't break upgrades?

Reply:
Sorry for the noise, it's a container name and likely not relevant for upgrade's concerns.